api-flask/docker-compose.develop.yml

```diff
@@ -1,19 +1,23 @@
-version: '3'
+version: '3.4'
 
 services:
   api:
     volumes:
       - ./app:/app
     ports:
-      - '80:80'
+      - "80:80"
+    build:
+        context: ./
+        dockerfile: ./Dockerfile
     environment:
       - FLASK_APP=main.py
       - FLASK_ENV=development
       - FLASK_DEBUG=1
       - 'RUN=flask run --host=0.0.0.0 --port=80'
+      - POSTGRES_HOST=alerting-db
       - POSTGRES_USER=postgres
       - POSTGRES_DB=postgres
-      - POSTGRES_PASSWORD='!ChangeMe!'
+      - POSTGRES_PASSWORD=!ChangeMe! # don't use '' to wrap string here
```

Comment:
no need for this comment, just remove the ' '

Reply:
It took me 2 hours to figure out why i can't connect to DB lol.

I will remove comment.

Reply:
Definitely better without it. Sorry about that!